lib/Widgets/ListItem.vala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2025 elementary, Inc. (https://elementary.io)
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+/**
+ * A standard list item widget for use in {@link Gtk.ListBox} and {@link Gtk.ListView}
+ *
+ * @since 7.7.0
+ */
+[Version (since = "7.7.0")]
+public class Granite.ListItem : Granite.Bin {
+    /**
+     * The main label for #this
+     */
+    public string text { get; set; }
+
+    /**
+     * Small, dim description text shown when active.
+     */
+    public string? description { get; set; }
+
+    class construct {
+        set_css_name ("granite-listitem");
```

Comment:
I don't know if there's a better way to name this. I wanted to avoid colliding with Gtk.ListItem which will be the parent of this widget in a ListView